tests/foreman/ui/test_ansible.py

```diff
@@ -76,19 +47,37 @@ def test_positive_create_variable_with_overrides(self, target_sat):
 
         :expectedresults: The variable is successfully created.
         """
+        user_cfg = admin_nailgun_config()
+        password = settings.server.admin_password
         key = gen_string('alpha')
+        param_type = 'integer'
+        if auth_type == 'non-admin':
+            ansible_manager_role = target_sat.api.Role().search(
+                query={'search': 'name="Ansible Roles Manager"'}
+            )
+            user = target_sat.api.User(
+                role=ansible_manager_role,
+                admin=True,
```

Comment:
Shouldn't `admin` be set to `False` for the non-admin user scenario?

Reply:
@synkd Good catch, updated